### DISCUSSION TITLE
MessageDialog Style

### DISCUSSION THREAD

**Discussion by estradege** [MahApps/MahApps.Metro#4493] [Q&A]
Hi everyone,
I would like to use the ShowMessageAsync function to display a confirmation for delete an item. The window is a small popup and the MessageDialog is not completely display :
![image](https://github.com/MahApps/MahApps.Metro/assets/61650659/a0cafecc-6ce3-4c1e-9c29-a56a20d22aee)

Is there a way to style the message dialog window and reduce the padding ?

I tried to add a ResourceDictionary but it doesn't work:

App.xaml :
```
<Application.Resources>
    <ResourceDictionary>
        <ResourceDictionary.MergedDictionaries>
            <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Controls.xaml" />
            <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Fonts.xaml" />
            <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Themes/Light.Blue.xaml" />
            <ResourceDictionary Source="/Styles/MessageDialog.xaml" />
        </ResourceDictionary.MergedDictionaries>
    </ResourceDictionary>
</Application.Resources>
```

MessageDialog.xaml :
```
<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                    xmlns:Dialogs="clr-namespace:MahApps.Metro.Controls.Dialogs;assembly=MahApps.Metro">
    <Style x:Key="CustomMessageDialog"
        BasedOn="{StaticResource {x:Type Dialogs:BaseMetroDialog}}"
        TargetType="{x:Type Dialogs:MessageDialog}">
        <Setter Property="Background" Value="Red"></Setter>
        <Setter Property="Opacity" Value=".2"></Setter>
        <Setter Property="Margin" Value="0"></Setter>
        <Setter Property="Padding" Value="0"></Setter>
    </Style>
</ResourceDictionary>

```

Thanks for your help,

**Comment by timunie** [comment]
I think you can just set these properties:
 https://github.com/MahApps/MahApps.Metro/blob/509c9d7e559025754854e4472db3b3adaaf9ef38/src/MahApps.Metro/Themes/Dialogs/MessageDialog.xaml#L23-L27

**Reply by estradege** [reply]
Thanks,
how can I set the DialogContentMargin property? The MessageDialog is open via the function ShowMessageAsync of MetroWindow class.

**Reply by punker76** [reply]
via the settings https://mahapps.github.io/mahapps.com/docs/dialogs/dialogsettings
